components/kyma-environment-broker/internal/cls/deprovisioner.go

```diff
@@ -38,7 +36,7 @@ type DeprovisionRequest struct {
 	Instance      servicemanager.InstanceKey
 }
 
-func (d *Deprovisioner) Deprovision(smClient servicemanager.Client, request *DeprovisionRequest) error {
+func (d *Deprovisioner) Deprovision(log logrus.FieldLogger, smClient servicemanager.Client, request *DeprovisionRequest) error {
```

Comment:
minor (could be fixed later): there is a convention in KEB that logger argument is the last one

Reply:
fixed